### PR TITLE
changed art gallery again to automated slideshow

### PR DESCRIPTION
**Pull request by christinachance**

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -16,22 +16,25 @@
...
-  for (i = 0; i < elements.length; i++) {
-    elements[i].style.flex = "25%";
-  }
+function showSlides(){
+    var i;
+    var slides = document.getElementsByClassName("mySlides");
+    for(i=0; i<slides.length;i++){
+        slides[i].style.display="none";
+    }
+    console.log(slides.length);
```
Comment (wulian1223) [inline portfolio/src/main/webapp/script.js:29]: removing debugging code?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -16,22 +16,25 @@
...
-var i;
 
-function four() {
-  for (i = 0; i < elements.length; i++) {
-    elements[i].style.flex = "25%";
-  }
+function showSlides(){
+    var i;
+    var slides = document.getElementsByClassName("mySlides");
+    for(i=0; i<slides.length;i++){
```
Comment (wulian1223) [inline portfolio/src/main/webapp/script.js:26]: use meaningful var name `slide` instead `i`

Comment (christinachance) [reply portfolio/src/main/webapp/script.js:26]: done.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (christinachance) [reply portfolio/src/main/webapp/script.js:29]: done.

**Comment by christinachance** [comment]
> In html files, could you remove redundant spaces for id and other attributes values, e.g.:
> img src = "images/art-4.jpg" alt = "Grandma's Living Room" style="height:100%"
> =>
> img src="images/art-4.jpg" alt="Grandma's Living Room" style="height:100%"

done.

**Pull request merged**